tests/server/test_calling_routes.py

```diff
@@ -485,6 +485,25 @@ def test_update_recording_name__returns_ok_if_recording_name_doesnt_exist(
     assert response.status_code == 200
 
 
+@pytest.mark.parametrize(
+    "test_new_file_name",
+    ["NormalName", "  TestName", "TestName  ", "  TestName  "],
+)
+def test_update_recording_name__trims_spaces_from_start_and_end_beta_2_mode(
```

Comment:
nitpick, you don't need to have beta_2_mode mentioned in the test name now that there is no beta 1 mode version of this test